### DISCUSSION TITLE
Support for OpenTelemetry tracing in the tracer?

### DISCUSSION THREAD

**Discussion by vkruoso** [moleculerjs/moleculer#1060] [Q&A] [answered]
Hi! 

I've added the tracing to moleculer. It instrument only the relevant moleculer parts, mainly service calls. I want to add extra instrumentation, like mysql2. This is provided by `@opentelemetry/instrumentation-mysql2`, for instance.

Is there any easy way to integrate both solutions?

Thanks!

**Comment by intech** [comment, marked as answer]
We discuss compatibility in [this thread](https://github.com/moleculerjs/moleculer/discussions/1062)
